src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonSynchronisationService.kt

```diff
@@ -144,11 +145,32 @@ class ContactPersonSynchronisationService(
   }
   suspend fun contactUpdated(event: ContactEvent) {
     val telemetry =
-      mapOf("offenderNo" to event.offenderIdDisplay, "bookingId" to event.bookingId, "personId" to event.personId, "contactId" to event.contactId)
-    telemetryClient.trackEvent(
-      "contactperson-contact-synchronisation-updated-success",
-      telemetry,
-    )
+      mutableMapOf(
+        "offenderNo" to event.offenderIdDisplay,
+        "bookingId" to event.bookingId,
+        "nomisPersonId" to event.personId,
+        "dpsContactId" to event.personId,
```

Comment:
just checking that the nomis person id is the ops contact id - but looks like it was intentional